docs/guide/setup.md

```diff
@@ -33,11 +33,11 @@ These are the possible options to pass when calling `VuexORM.use()`:
 - `debug` (optional, default: `false`): Set to true to activate the debug logging.
 - `url` (optional, default: `/graphql`): The URL to the graphql api. Will be passed to apollo-client.
 - `headers` (optional, default: `{}`) HTTP Headers. See
-  [apollo-link-http])(https://github.com/apollographql/apollo-link/tree/master/packages/apollo-link-http#options).
+  [apollo-link-http](https://github.com/apollographql/apollo-link/tree/master/packages/apollo-link-http#options).
   This can be a static object or a function, returning a object, which will be called before a request is made.
-- `credentials` (optional, default: `same-origin`) Credentials Policy. See [apollo-link-http])(https://github.com/apollographql/apollo-link/tree/master/packages/apollo-link-http#options)
-- `useGETForQueries` (optional, default: `false`) Use GET for queries (not for mutations). See [apollo-link-http])(https://github.com/apollographql/apollo-link/tree/master/packages/apollo-link-http#options)
-- `connectionQueryMode` (optional, default: `auto`). One of `auto | nodes | edges | plain`. See [Connection Mode](connection-mode.md)
+- `credentials` (optional, default: `same-origin`) Credentials Policy. See [apollo-link-http](https://github.com/apollographql/apollo-link/tree/master/packages/apollo-link-http#options)
+- `useGETForQueries` (optional, default: `false`) Use GET for queries (not for mutations). See [apollo-link-http](https://github.com/apollographql/apollo-link/tree/master/packages/apollo-link-http#options)
+- `connectionQueryMode` (optional, default: `auto`). One of `auto | nodes | edges | plain`. See [Connection Mode](/plugin-graphql/guide/connection-mode.html)
```

Comment:
The original version should work according to https://vuepress.vuejs.org/guide/markdown.html#links.

But it doesn't 🤔

Reply:
I fixed this in another branch.